addons/modules/fnc_moduleDefend.sqf

```diff
@@ -0,0 +1,76 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_moduleAttack
+
+Description:
+    A function for commanding a group to defend a location with information
+    parsed from a module.
+
+Parameters:
+    - Group (Array of Classnames or an array of a config)
+    - Spawn Point (XYZ, Object, Location or Marker)
+    - Attack Point (XYZ, Object, Location, Group, or Marker)
+
+Optional:
+    - Attack Radius (Number)
+
+Example:
+    (begin example)
+    (end)
+
+Returns:
+    Nil
+
+Author:
+    WiredTiger
+
+---------------------------------------------------------------------------- */
+
+//SCRIPT(moduleDefend);
+
+private [
+    "_logic",
+    "_units",
+    "_localUnits",
+    "_defendPos",
+    "_defendLocType",
+    "_defendRadius",
+    "_defendSetPos",
+    "_threshold",
+    "_canPatrol"
+];
+
+// Only server, dedicated, or headless beyond this point
+if (hasInterface && !isServer) exitWith {};
+
+_units = param [1,[],[[]]];
+_localUnits = [];
+
+{
+    // Find owner of unit if headless client is present
+    if (local _x) then {
+        _localUnits pushBack _x;
+    };
+} forEach _units;
+if (count _localUnits == 0) exitWith {};
+
```

Comment:
@jonpas I just want to make sure I am not misunderstanding this. Cuel's suggestion would NOT work on linux builds? Therefore, I should keep the code how it is.